test/e2e/lib/gutenberg/tracking/general-tests.js

```diff
@@ -158,6 +158,23 @@ export function createGeneralTests( { it, editorType, postType } ) {
 		);
 	} );
 
+	it( `W button should not trigger the "wpcom_block_editor_close_click" event when it's connected to a sidebar`, async function () {
+		const editor = await EditorComponent.Expect( this.driver, gutenbergEditorType );
+
+		await editor.toggleNavigationSidebar();
+		await editor.toggleNavigationSidebar();
```

Comment:
Why a double call here?

Reply:
To make sure we don't leave it open for the next test. And it's also to make sure the event isn't triggered for both open and close action.

Reply:
Ah, gotcha. A short explanation comment would be nice then! 🙏